### ISSUE TITLE
navigation active item class not applied when list items are not inside v-list-group

### ISSUE THREAD

**Issue by jacekkarczmarczyk** [vuetifyjs/vuetify#1374]
### Steps to reproduce

Create drawer with navigation items that are not inside v-list-group component

### Versions
vuejs 2.4.2
vue router 2.7.0
vuetify 0.14.11

### What is expected ?
Current route item should have primary color

### What is actually happening ?
Current route item has default (dark grey) color

### Reproduction Link

Without v-list-group
https://codepen.io/anon/pen/ayEOJG

With v-list-group
https://codepen.io/anon/pen/jLYPaB

As far as I'm concerned it's a regression from 0.14.8 or 0.14.9, not sure, anyway it was working before (although only the icon got the primary color, with 0.14.11 text gets primary color, but not the icon)

**Comment by atilkan** [comment]
@johnleider not on the document.

**Comment by tundebabzy** [comment]
@atilkan I also just did notice it but I've fixed it. Here are the steps I took:
- This bug was fixed in bc52af9 but was just released yesterday in v1.3.1 so make sure you are on the latest release.
- Instead of using `@click` in `v-list-title` component for navigation, use the `to` prop

After taking these steps, I was able to get the desired behavior. I wish I could add this to the documentation, but I actually have no idea where to start
